examples/test_api.c

```diff
@@ -185,11 +190,36 @@ int test_abtoull(char* str, unsigned long long* val)
   return SCR_SUCCESS;
 }
 
-double getbw(char* name, char* buf, size_t size, int times)
+size_t get_my_shared_file_offset()
+{
+  size_t file_offset;
+  size_t send_buf;
```

Comment:
To be safe, we should make these types precisely match the type we give to MPI.  We can cast back to size_t after the MPI communication.
```suggestion
  uint64_t file_offset;
  uint64_t send_buf;
```